src/leaf.ml

```diff
@@ -37,7 +37,9 @@ functor
 
     let will_overflow t = LeafFmt.length t.leaf >= 2 * Params.fanout
 
-    let underflow t = LeafFmt.length t.leaf < Params.fanout
+    let underflow t =
```

Comment:
Maybe `vacancy`?

Reply:
Why do you not like `underflow` ?

Reply:
To me, `underflow` designates a "bad" situation, just like `overflow` (e.g. [in c](https://en.cppreference.com/w/cpp/numeric/fenv/FE_exceptions)). 

In your case, `LeafFmt.length t.leaf < Params.fanout` is the normal case, `LeafFmt.length t.leaf > 2 * Params.fanout` is the exceptional case.

But that's mainly a question of taste :)

Reply:
`underflow` is in fact a bad situation, the normal case is when `2 * Params.fanout >= LeafFmt.length t.leaf >= Params.fanout`

Reply:
oh sorry, my mistake